### DISCUSSION TITLE
Is Vuetify 3 (next) free for commercial use?

### DISCUSSION THREAD

**Discussion by nshadx** [vuetifyjs/vuetify#15652] [Q&A] [answered]
I have a question in a title

**Comment by KaelWD** [comment, marked as answer]
https://github.com/vuetifyjs/vuetify/blob/next/LICENSE.md
![Screenshot_20220818_214425](https://user-images.githubusercontent.com/16421948/185386846-fb30f7a6-35d6-455e-9aa2-cdd495b4d0b2.png)
